src/background/utils/extension-api.ts

```diff
@@ -40,16 +40,24 @@ export function canInjectScript(url: string) {
 let isWriting = false;
 
 export async function readSyncStorage<T extends {[key: string]: any}>(defaults: T): Promise<T> {
-    return new Promise<T>((resolve) => {
+    return new Promise<T>((resolve, reject) => {
         chrome.storage.sync.get(defaults, (sync: T) => {
+            if (chrome.runtime.lastError) {
```

Comment:
What if there will be `chrome.runtime.lastError`? It will throw an exception and the extension won't start.

Maybe log an error and return defaults?